src/main/java/net/openhft/chronicle/network/AlwaysStartOnPrimaryConnectionStrategy.java

```diff
@@ -51,6 +51,9 @@
     private int socketConnectionTimeoutMs = Integer.getInteger("connectionStrategy.socketConnectionTimeoutMs", 1);
     private long pauseMillisBeforeReconnect = Integer.getInteger("connectionStrategy.pauseMillisBeforeReconnect", 500);
     private ClientConnectionMonitor clientConnectionMonitor = new VanillaClientConnectionMonitor();
+    private long minTimeSec = Integer.getInteger("connectionStrategy.pause.min.secs", 5);
```

Comment:
This property name introduces yet another naming scheme to the configuration properties; consider using one of the existing, e.g., `connectionStrategy.minimumPauseSeconds` and `connectionStrategy.maximumPauseSeconds`?

And perhaps open a new issue to address the `Ms` vs `Millis` and lack of units in the other property names.

Reply:
@JanStureNielsen these property names are legacy, and in use by customers

Reply:
> This property name introduces yet another naming scheme to the configuration properties; consider using one of the existing, e.g., `connectionStrategy.minimumPauseSeconds` and `connectionStrategy.maximumPauseSeconds`?
> 
> And perhaps open a new issue to address the `Ms` vs `Millis` and lack of units in the other property names.

I agree - these new suggested method names are better

Reply:
> > This property name introduces yet another naming scheme to the configuration properties; consider using one of the existing, e.g., `connectionStrategy.minimumPauseSeconds` and `connectionStrategy.maximumPauseSeconds`?
> > And perhaps open a new issue to address the `Ms` vs `Millis` and lack of units in the other property names.
> 
> I agree - these new suggested method names are better

Yes but that is changing property names that customers are using. If we change then we need to support the old for a period. If we are going to change we should review all the system properties and consistentise them